### DISCUSSION TITLE
Estimated print time

### DISCUSSION THREAD

**Discussion by drveni** [supermerill/SuperSlicer#1960] [Q&A]
Hi all

Could someone please help me. 

For my estimate printing time. Im slice and print time is 7h30min. When i print in klipper, my printer print this file for 2h10min.
what i'm doing wrong? how to slice file what setings i need to have to have correct estimate time for print file. it doesn't need be 100%. but 7h and 2h are big difference. what im doing wrong?

**Comment by neophyl** [comment]
Normally you would set all the machine limit values  in printer settings to match those your machine is actually running at (or close enough) and then select for time estimate in the drop down. That way the values are used for the estimates.

**Reply by drveni** [reply]
> Normally you would set all the machine limit values in printer settings to match those your machine is actually running at (or close enough) and then select for time estimate in the drop down. That way the values are used for the estimates.

thx for replay... and that is the problem. in super slicer im emit from drop down menu to "emit to print estimate" and still says over 7h print. no matter what i changed.. same result. when i choose marlin, or klipper.. emit or not emit to print file. is that bug? im on mac.?

**Reply by neophyl** [reply]
It may be a bug but would it be possible for you to attach a zipped up project file (3mf] that has the problem so we can take a look please. 
If there is an issue then Merrill will need the same to figure it out anyway.

**Reply by supermerill** [reply]
"emit to print estimate" ?  you means "Use also for print estimate" ?

If all your printer limits are good, then you can set the `time estimation compensation` setting to 29%

**Reply by drveni** [reply]
yes, I tested, at 29%. i still have a weird problem.
at a print speed of 80mm then it is exactly about 5 min print time. when I set the speed to 50mm ... he calculated that the print time was about 2 hours. he printed over 4. 

and when I change the settings. the lower part is shaded for entering or changing the acceleration value(cant change values). I have to get out of the program and restart it again to take off my darkened part. and one can only make changes. looks like a bug to me .?!

**Reply by supermerill** [reply]
> when I change the settings. the lower part is shaded

The text is grey? if it's that, it's to indicate what field is used for 'emit limits to gcode'. Seems buggy, I will fix that, but it's only coloring.

> cant change values

unless it's on 'disable', I never encountered this problem.
